ansible/configs/ocp4-disconnected-osp-lab/files/clouds.yaml.j2

```diff
@@ -2,11 +2,16 @@ clouds:
   {{ osp_project_name }}:
     auth:
       auth_url: "{{ osp_auth_url }}"
+{% if osp_project_create | bool %}
       username: "{{ guid }}-user"
+      password: "{{ hostvars['localhost']['heat_user_password'] }}"
+{% else %}
+      username: "{{ osp_auth_username_member }}"
+      password: "{{ osp_auth_password_member }}"
+{% endif %}
       project_name: "{{ osp_project_name }}"
-      project_id: "{{ hostvars['localhost']['osp_project_info'][0].id }}"
+      project_id: "{{ hostvars['localhost']['osp_project_info'][0].id | d(osp_project_id) }}"
```

Comment:
please change this to `default` instead of `d`

Reply:
Done